t/node/upstream-websocket.t

```diff
@@ -33,16 +33,15 @@ __DATA__
             local t = require("lib.test_admin").test
             local code, body = t('/apisix/admin/routes/1',
                  ngx.HTTP_PUT,
-                 [[{
-                        "upstream": {
-                            "nodes": {
-                                "127.0.0.1:1980": 1
-                            },
-                            "enable_websocket": true,
-                            "type": "roundrobin"
+                [[{
+                    "upstream": {
+                        "nodes": {
+                            "127.0.0.1:1980": 1
                         },
-                        "uri": "/websocket_handshake"
-                }]]
```

Comment:
missing `}]]`

Reply:
I've fixed it,  please review again :) @membphis